src/main/java/dk/cngroup/lentils/service/HintService.java

```diff
@@ -50,4 +57,17 @@ public void deleteById(final Long id) {
     public void deleteAlHintsByCypher(final Cypher cypher) {
         hintRepository.deleteByCypher(cypher);
     }
+
+    public List<Hint> getHintsNotTakenByTeam(final Cypher cypher, final Team team) {
+        List<Hint> allHintsByCypher = getAllByCypher(cypher);
+        List<HintTaken> allHintsTakenByTeam = hintTakenService.getAllByTeam(team);
+        List<Long> allHintsTakenByTeamHintIds = new ArrayList<>();
```

Comment:
Toto by slo krasne nahradit streamem
```java
List<Long> allHintsTakenByTeamHintIds = allHintsTakenByTeam.stream()
    .map(ht -> ht.hetHint().getId))
    .collect(Collectors.toList())
```
Nebo nejak tak by to melo jit (na mobilu se to blbe pise..)